packages/mobile/src/analytics/Properties.tsx

```diff
@@ -779,8 +779,7 @@ interface FiatExchangeEventsProperties {
     provider: string
   }
   [FiatExchangeEvents.cash_in_success]: {
-    provider: string
-    currency: string
```

Comment:
is currency not useful anymore? seems good to have maybe

Reply:
The issue is that it's actually not possible to get currency. The way we were calling this event before (which allowed for logging currency) is not very robust and will be deprecated